app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -592,6 +609,82 @@ public boolean isMultiWindowEnabled() {
         }
     }
 
+    public boolean isShowSearchResultOnCancel() {
+        return getBool(R.string.pref_key__is_show_search_result_on_cancel_enabled, true);
```

Comment:
Note: We can use backup-restore to change these values (if necessary for testing etc)